crates/bitwarden-exporters/src/lib.rs

```diff
@@ -212,14 +212,14 @@ pub struct Login {
 }
 
 #[allow(missing_docs)]
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
 pub struct LoginUri {
     pub uri: Option<String>,
     pub r#match: Option<u8>,
 }
 
 #[allow(missing_docs)]
-#[derive(Clone)]
+#[derive(Debug, Clone, PartialEq, Eq)]
```

Comment:
question: are these plain-text fields? If yes then:

suggestion: make equality derivation (and Debug for that matter) isolated to tests.

Just as a precaution, we shouldn't use the default equality checker for things that contain crypto keys. This probably won't ever end up in a place where it can be exploited, but still 🤷